docker-entrypoint.sh

```diff
@@ -16,5 +16,13 @@ fi
 echo "starting mediatree import app"
 python quotaclimat/data_processing/mediatree/api_import.py
 
-echo "apply dbt models full-refresh"
-poetry run dbt run --full-refresh # drop and recreate materialized views with the most up-to-date data.
\ No newline at end of file
+echo "apply dbt models"
+
+     # drop and recreate materialized views with the most up-to-date data.
+
+echo "apply dbt incremental model core_query_causal_links for each month"
+for m in {2022-01..2025-09}; do
```

Comment:
after 1st run, we'll switch to just poetry run dbt run -m core_query_causal_links